js/ui/windowManager.js

```diff
@@ -658,6 +630,8 @@ const WindowManager = new Lang.Class({
             this._destroyWindowDone(shellwm, actor);
             this._rotateInCompleted(actor);
             this._rotateOutCompleted(actor);
+            this._codingManager.rotateInCompleted(actor);
+            this._codingManager.rotateOutCompleted(actor);
```

Comment:
Could make `this._rotateInCompled` call the relevant function on `this._codingManager` too.